frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseForcePushBranchAction.java

```diff
@@ -70,15 +70,15 @@ public void update(AnActionEvent anActionEvent) {
     if (isEnabled) {
 
       if (getCurrentBranchNameIfManaged(anActionEvent).equals(branchName)) {
-        presentation.setText("Push Current Branch");
+        presentation.setText("Force Push Current Branch");
```

Comment:
Hmmm maybe just `Push` for `AheadOfRemote`/`Untracked`; would be nice if the default action would be a regular push, not force push for those states